.github/workflows/codecov.yml

```diff
@@ -18,13 +18,22 @@ jobs:
   # This workflow contains a single job called "build"
   build:
     # The type of runner that the job will run on
-    runs-on: eclipse/kiso-build-env:v0.2.1
+    runs-on: ubuntu-latest
 
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
+      # Pre requisite installation
+      - name: Prerequisite installation
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
```

Comment:
`python-version: '3.10`

Reply:
Yea ok, I probably need to maintain the compatibility with 3.7 via the Eclipse Jenkins right?